src/Goteo/Model/Project.php

```diff
@@ -2506,9 +2506,10 @@ public function getValidation() {
             } else {
                 $res->rewards = 100;
             }
-            if($total < 3) {
+            $rewards_required = Config::get('rewards.required') ?: 3;
+            if($total < $rewards_required) {
                 $errors['rewards'][] = 'rewards_required';
-                $res->rewards *= $total / 3;
+                $res->rewards *= $total / $rewards_required;
```

Comment:
Beware: this could fail when $rewards_required is 0.

Reply:
Absolutely right. @alexm could you put a conditional to bypass checks in that case?

Reply:
Please, note that `"0" ?: 3` expression returns `3`.

On the other side, `" " ?: 3` returns space, so it would be possible to have a divide by zero error in that case. But, how probable is that anybody uses a space as a value for that setting?

Anyway, I guess I can improve it like this and it should work for all cases:

```
$rewards_required = intval(Config::get("rewards.required")) ?: 3;
```